cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -47,6 +47,8 @@
 {% endif -%}
 {% endif -%}
 
+{% set params = "--master=127.0.0.1:8080" + " " + machines + " " + cluster_name + " " + minion_regexp + " " + cloud_provider  + " " + sync_nodes + " " + cloud_config + " " + pillar['log_level'] -%}
```

Comment:
we always had --master=127.0.0.1:8080. check linke 62 of the checked in version (on left side).